scripts/build-tools.sh

```diff
@@ -5,69 +5,106 @@
 # fail immediately on any errors
 set -e
 
-
 print_usage()
 {
         cat <<EOFUSAGE
-usage: $0 [-t|-f]
+usage: $0 [-c|-f|-h|-l|-p|-t|-T]
+       [-c] Build ctl
+       [-f] Build fuzzer
+       [-h] Display help
+       [-l] Build logger
+       [-p] Build probes
        [-t] Build test topologies
-       [-f] Build fuzzer"
+       [-T] Build topologies
 EOFUSAGE
 }
 
```

Comment:
Can we compile everything if the script has no parameters?

Reply:
yes, it compiles everything if no parameters at https://github.com/thesofproject/sof/pull/3064/files#diff-b3893e3cc0c1ad18c233c8ccb11447ecR55